kubernetes/v1alpha1/spec.go

```diff
@@ -58,11 +58,17 @@ type MonitoringDashboardChart struct {
 	Min          *int                             `json:"min"`
 	Max          *int                             `json:"max"`
 	MetricName   string                           `json:"metricName"`
+	Metrics      []MonitoringDashboardMetric      `json:"metrics"`
```

Comment:
MonitoringDashboardChart has a field called metricName (in json) terminology.
Now it has also an array field as "metrics" which is a list of metricsName/displayName.

From external point of view it makes model less clear.
My first understanding is there is a main metric and the other are optional/subordinated.

I'm not sure if I'm understanding correctly what it represents. If MonitoringDashboardChart is going to support 1:N metrics I'd expect to see them all in a single place (and having an array of length == 1 for single case, I think).

Reply:
This is correct to say that there's all in a single place and metrics array of size 1 does represent the old case. This is actually what happens in the code.

But `metricName` is kept for backward compatibility, and its content is used to fill the `metrics` array. I will explain that in the documentation (in kiali.io), and mark that as deprecated. This is a transitional state

Reply:
I've added a comment to mark metricName Deprecated